server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -546,40 +507,42 @@ private String createDN(Consumer consumer, Owner owner) {
         return result;
     }
 
-    private byte[] createContentAccessDataPayload(Owner owner, Environment environment) throws IOException {
-        // fake a product dto as a container for the org content
-        Set<Product> containerSet = new HashSet<>();
-        Set<String> entitledProductIds = new HashSet<>();
-        List<org.candlepin.model.dto.Product> productModels = new ArrayList<>();
-        Map<String, EnvironmentContent> promotedContent = getPromotedContent(environment);
-        String contentPrefix = this.getContentPrefix(owner, environment);
+    private byte[] createContentAccessDataPayload(Owner owner, Environment environment, Consumer consumer)
+        throws IOException {
         Product container = new Product();
-        Entitlement emptyEnt = new Entitlement();
-        Pool emptyPool = new Pool();
-        Product skuProduct = new Product();
-        Consumer emptyConsumer = new Consumer();
-        emptyConsumer.setOwner(owner);
-
-        containerSet.add(container);
         container.setId("content_access");
         container.setName(" Content Access");
 
         this.ownerContentCurator.getActiveContentByOwner(owner.getId())
-            .forEach((content, enabled) -> container.addContent(content, enabled));
+            .forEach(container::addContent);
 
+        Consumer emptyConsumer = new Consumer();
```

Comment:
Now that we're passing the consumer in, and these certs are generated per-consumer, do we need to generate a fake consumer here at all?

Reply:
That's true. We don't need a fake one anymore since we have a real one. Fixed.